src/domain/registration/state/RegistrationReducers.tsx

```diff
@@ -12,10 +12,9 @@ export const defaultRegistrationData: RegistrationData = {
   formValues: {
     children: [
       {
-        birthdate: '',
-        firstName: '',
+        birthyear: new Date().getFullYear(),
```

Comment:
It's debatable whether it's really good to have the current year here as a default value. I'm vote for not having a default value here at all as it is always incorrect.